docs/xml/elements/contest_base.rst

```diff
@@ -1,37 +1,103 @@
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| ElectorateSpecification |:doc:`InternationalizedText      | Optional  | Single   |Specifies any changes to  |If the element is     |
+|                         |<internationalized_text>`        |           |          |the eligible electorate   |invalid or not        |
+|                         |                                 |           |          |for this contest past the |present, the          |
+|                         |                                 |           |          |usual, "all registered    |implementation should |
+|                         |                                 |           |          |voters" electorate. This  |ignore it.            |
+|                         |                                 |           |          |subtag will most often be |                      |
+|                         |                                 |           |          |used for primaries and    |                      |
+|                         |                                 |           |          |local elections. In       |                      |
+|                         |                                 |           |          |primaries, voters may have|                      |
+|                         |                                 |           |          |to be registered as a     |                      |
+|                         |                                 |           |          |specific party to vote, or|                      |
+|                         |                                 |           |          |there may be special rules|                      |
+|                         |                                 |           |          |for which ballot a voter  |                      |
+|                         |                                 |           |          |can pull. In some local   |                      |
+|                         |                                 |           |          |elections, non-citizens   |                      |
+|                         |                                 |           |          |can vote.                 |                      |
+|                         |                                 |           |          |                          |                      |
+|                         |                                 |           |          |                          |                      |
+|                         |                                 |           |          |                          |                      |
++-------------------------+---------------------------------+-----------+----------+--------------------------+----------------------+
+| ExternalIdentifiers     |:doc:`ExternalIdentifiers        | Optional  | Single   |Another identifier for a  |If the element is     |
```

Comment:
`ExternalIdentifiers` actually supports multiple third-party IDs, so this should be "Other identifiers" instead of "Another identifier".